packages/react/src/components/Forms/TextArea/index.scss

```diff
@@ -0,0 +1,97 @@
+.c__textarea__wrapper {
+  border-radius: var(--c--components--forms-textarea--border-radius);
+  border-width: var(--c--components--forms-textarea--border-width);
+  border-color: var(--c--components--forms-textarea--border-color);
+  border-style: var(--c--components--forms-textarea--border-style);
+  background-color: var(--c--components--forms-textarea--background-color);
+  transition: border var(--c--theme--transitions--duration) var(--c--theme--transitions--ease-out);
+  overflow: hidden;
+
+  .labelled-box__label {
+    padding: 0 1rem;
+  }
+
+  .c__textarea {
+    border: none;
+    font-size: var(--c--components--forms-textarea--font-size);
+    font-weight: var(--c--components--forms-textarea--font-weight);
+    width: 100% !important;
+    padding: 0 1rem;
+    font-family: var(--c--theme--font--families--base);
+
+    &:focus-visible {
+      outline: none;
+    }
+  }
+
+  &:hover {
+    border-radius: var(--c--components--forms-textarea--border-radius--hover);
+    border-color: var(--c--components--forms-textarea--border-color--hover);
+  }
+
+  &:focus-within {
+    border-radius: var(--c--components--forms-textarea--border-radius--focus);
+    border-color: var(--c--components--forms-textarea--border-color--focus) !important;
```

Comment:
Ditto

Reply:
I took the same approach we have in the other components such as `Input` so that focus states turns the border blue no matter what even when the component is in success or error mode ( green or red borders ) - it avoids having to artificially define more specific class selectors and keeps `&:focus-within` as simple as it is. 

But I get that's not perfect :/